systemd.conf

```diff
@@ -0,0 +1,13 @@
+# shellinabox.service
```

Comment:
upstart != systemd

But yeah, lets not leave redundant stuff around like this. It's just going to get dusty and outdated.